students/hyeonwoo/users/views.py

```diff
@@ -1,3 +1,65 @@
-from django.shortcuts import render
+import json
+from django.core.exceptions import ValidationError
+
+from django.shortcuts   import render
+from django.http        import JsonResponse
+from django.views       import View
+
+from .validation        import validation_email, validation_password
+from .models            import User
+
+class SignUp(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try :
+            first_name    = data['first_name']
```

Comment:
data도 try문 안으로 넣어 주세요.
```suggestion
        try :
            data = json.loads(request.body)
            first_name    = data['first_name']
```

Reply:
네 알겠습니다. 수정 후 다시 PR 하도록 하겠습니다.